content/docs/getting-started.md

```diff
@@ -18,109 +18,109 @@ redirect_from:
+## Ucz się Reacta {#learn-react}
 
-People come to React from different backgrounds and with different learning styles. Whether you prefer a more theoretical or a practical approach, we hope you'll find this section helpful.
+Ludzie zaczynają przygodę z Reactem, przychodząc z różnych środowisk i doświadczeń z językami programowania. Niezależnie czy preferujesz bardziej techniczne, czy praktyczne podejście, mamy nadzieję, że uznasz tę sekcję za pomocną.
 
-* If you prefer to **learn by doing**, start with our [practical tutorial](/tutorial/tutorial.html).
-* If you prefer to **learn concepts step by step**, start with our [guide to main concepts](/docs/hello-world.html).
+* Jeśli preferujesz **naukę przez praktykę**, zacznij z naszymi [praktycznymi samouczkami](/tutorial/tutorial.html).
+* Jeśli wolisz **naukę pojęć krok po kroku**, zacznij od [przewodnika po głównych pojęciach](/docs/hello-world.html).
 
-Like any unfamiliar technology, React does have a learning curve. With practice and some patience, you *will* get the hang of it.
+Jak każda nowa technologia, z którą spotykasz się pierwszy raz, React także ma swoją krzywą uczenia. Przy odrobinie praktyki i cierpliwości *zrozumiesz* na czym polega.
 
-### First Examples {#first-examples}
+### Podstawowe przykłady {#first-examples}
 
-The [React homepage](/) contains a few small React examples with a live editor. Even if you don't know anything about React yet, try changing their code and see how it affects the result.
+[Strona główna](/) Reacta zawiera kilka małych przykładów z edytorem kodu live. Nawet jeśli nie wiesz jeszcze nic o Reakcie, spróbuj zmienić kod i zobaczyć, jakie to przyniesie efekty.
 
-### React for Beginners {#react-for-beginners}
+### React dla początkujących {#react-for-beginners}
 
-If you feel that the React documentation goes at a faster pace than you're comfortable with, check out [this overview of React by Tania Rascia](https://www.taniarascia.com/getting-started-with-react/). It introduces the most important React concepts in a detailed, beginner-friendly way. Once you're done, give the documentation another try!
+Jeśli wydaje Ci się, że dokumentacja Reacta prowadzi Cię w zbyt szybkim tempie, sprawdz [ten przegląd Reacta napisany przez Tanię Rascię](https://www.taniarascia.com/getting-started-with-react/). Wprowadzi Cię on szczegółowo w najistotniejsze zagadnienia Reacta w szczegółach, zachowując styl zrozumiały dla początkujących. Gdy już skończysz, spróbuj ponownie przeczytać naszą dokumentację!
 
-### React for Designers {#react-for-designers}
+### React dla Designerów {#react-for-designers}
 
-If you're coming from a design background, [these resources](http://reactfordesigners.com/) are a great place to get started.
+Jeśli zaczynając z Reactem przychodzisz ze środowiska graficznego, [te źródła](http://reactfordesigners.com/) mogą być świetnym punktem startowym.
 
-### JavaScript Resources {#javascript-resources}
+### Źródła JavaScript {#javascript-resources}
```

Comment:
```suggestion
### Kod źródłowy w JavaScripcie {#javascript-resources}
```

Reply:
No tutaj się nie zgadzam z poprawką. Resource to nie jest kod źródłowy. Źródło może nie będzie idealnym słowem, ale Kod źródłowy jest zdecydowanie mylące.

Reply:
Ok, mnie tutaj bardziej się nie podoba "Źródła *JavaScript*". Może "Źródła javascriptowe"? Napisane w JavaScripcie? Bo w obecnej formie to jest tylko nic nie znaczący zlepek słów.